content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,228 @@
++++
+title = "Train and deploy on GCP from a local notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform from a local notebook."
+weight = 30
++++
+
+This guide introduces you to using Kubeflow Fairing to train and deploy a
+model to Kubeflow on Google Kubernetes Engine, and Google Cloud ML Engine.
```

Comment:
Google Kubernetes Engine (GKE)

For later usages on this page, you can just use "GKE".

(GKE has been approved as a product name abbreviation, and is well known amongst our audience.)

Reply:
Done